documentation/whatsnew/freemarker/freemarker-news-4.3.0.Beta2.adoc

```diff
@@ -7,6 +7,6 @@
 
 == Include Directive Hyperlink
 
-Freemarker editor now shows hyperlink for #include directive with absolute path to make navigation through complex templates easier. #include directive with relative path and accusition is not supported yet. 
+Freemarker editor now shows hyperlink for #include directives wich references a file in the same directory. #include directive with relative/absolute path and accusition is not supported yet. 
```

Comment:
@dgolovin you have a typo here: wich -> which; and what is accusition? My dictionary won't find it.